examples/CQL/README.md

```diff
@@ -0,0 +1,35 @@
+## Reproduce CQL with PARL
+
+Different from Online RL which gather new data directly from the environment, Offline RL algorithms learn a policy or Q-function from static dataset, without additional interaction with the environment.
+
+Based on PARL, the CQL algorithm of deep reinforcement learning has been reproduced, reaching the same level of indicators as the paper on continuous control datasets from the D4RL benchmark.
+
+> Paper: CQL in [Conservative Q-Learning for Offline Reinforcement Learning](https://arxiv.org/abs/2006.04779)
+
+### Env and dataset introduction
++ D4RL datasets: Please see [here](https://sites.google.com/view/d4rl/home) to know more about D4RL datasets. D4RL require Mujoco as a dependency. For more D4RL usage methods, please refer to its [guide](https://github.com/rail-berkeley/d4rl#using-d4rl).
```

Comment:
The algorithm is tested in the [D4RL dataset](https://sites.google.com/view/d4rl/home), one of the most commonly used dataset for offline RL. For more details about D4RL usagage, please refer to [the page](https://github.com/rail-berkeley/d4rl#using-d4rl).